api.web.yaml

```diff
@@ -275,12 +278,7 @@ components:
         agreements:
           $ref: "#/components/schemas/ProfileAgreements"
 
-    People_AdminResponse:
-      type: object
-      required:
-        - people
-      properties:
-        people:
-          type: array
-          items:
-            $ref: "#/components/schemas/Person"
+    People:
+      type: array
+      items:
+        $ref: "#/components/schemas/Person"
```

Comment:
little simplification here because I'd expect that we'll return the same data for approved researchers who 'own' people, just not everyone